packages/ruby/spec/readme/har_collection_spec.rb

```diff
@@ -0,0 +1,77 @@
+require "readme/har_collection"
+require "readme/filter"
+
+RSpec.describe Readme::HarCollection do
+  describe "#to_h" do
+    context "with FilterParams filter" do
+      it "returns the given hash without the given filter values" do
+        filter_values = ["filter"]
+        filter = Filter::FilterParams.new(filter_values)
+        hash = {"filter" => "filtered", "dont_filter" => "not filtered"}
```

Comment:
Thoughts on giving these names like "keep" and "reject"? At some point too many variables have the name "filter" in them and it's hard to keep them straight 😅

Reply:
Heh can do.